src/auth-service/utils/group.util.js

```
@@ -230,7 +230,20 @@ const groupUtil = {
               joined: {
                 $dateToString: {
                   format: "%Y-%m-%d",
-                  date: { $ifNull: ["$group_roles.createdAt", "$_id"] },
+                  date: {
+                    $ifNull: [
+                      {
+                        $cond: {
+                          if: {
+                            $eq: [{ $type: "$group_roles.createdAt" }, "date"],
+                          },
+                          then: "$group_roles.createdAt",
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)